src/lib.rs

```diff
@@ -98,6 +98,7 @@ pub use message::{message_prettify, DEFAULT_COMMENT_CHAR};
 pub use note::{Note, Notes};
 pub use object::Object;
 pub use oid::Oid;
+pub use packbuilder::PackBuilder;
```

Comment:
Would it make sense to rename it to something like `PackBuilderStage` or `PackingStage` in then?